doc/LTI_1.3_PLATFORM.md

```diff
@@ -0,0 +1,124 @@
+[OAuth2](https://www.rfc-editor.org/rfc/rfc6749.html) and [OpenID
+Connect](https://openid.net/specs/openid-connect-core-1_0.html) specifications,
+and the A+ LTI 1.3 platform implementation therefore uses the
+[oauthlib](https://pypi.org/project/oauthlib/) for implementing the core
+security features.
+
+
+## Configuring a new LTI 1.3 Tool in A+
+
+In order to take a new LTI 1.3 Tool into use, sysadmin needs to configure the
+following settings in the Admin view, under *External_Services / LTI v1.3
+services*. These settings should be provided by the tool provider.
+
+* **URL** is the LTI launch URL from which the service is triggered.
+
+* **Destination region** and **Privacy notice URL** are as with other external
+  services, depending on whether the service is internal, inside EU or outside
+  EU.
+
+* **Menu label** and **Menu icon class** are visible in the side menu, if
+  configured. The menu label is also referred to when configuring the LTI-based
+  exercises.
+
+* **Login initiation URL** is used when A+ LTI 1.3 platform implementation
+  starts the authentication handshake with tool, as specified in [Section
+  5.1](https://www.imsglobal.org/spec/security/v1p0/#platform-originating-messages)
+  of the Security Framework specification. The tool provider should provide the
+  correct URL to use.
+
+* **Client ID** identifies the tool in the LTI messages and should be unique for
+  each tool configuration. A+ allows the system admin to select the identifier,
+  and the exact format of the identifier is not limited in any way, as long as
+  it is unique.
+
+* **Deployment ID** allows separating different deployments for a particular
+  client. Currently A+ does not have any functionality to support different
+  deployments within a single client, but the configuration is included for
+  protocol compliance, and for possible future use. The format of the deployment
+  ID is free for the system admin to choose.
+
+* **JWKS URL** is the address from which the tool's public key can found in
+  [JSON Web Key (JWK) format](https://www.rfc-editor.org/rfc/rfc7517.html). The
+  tool administrators will provide this URL.
+
+* **Share launcher's name, email and student ID** option controls whether A+ shares
+  user's real name, E-mail and student ID with the tool.
+
+
+## LTI 1.3 configuration data shared with the tool provider
+
+Also the LTI tool administrator needs to configure some details about A+ at the
+tool's end. Typically, the following information is needed (the exact
+terminology may vary depending on the tool):
+
+* **Issuer (or platform ID)**: This the issuer identifier, used in JWT tokens in
+  LTI exchanges. With A+ this should be equal to the value of BASE_URL settings
+  variable, for example "https://plus.cs.aalto.fi" in Aalto's production
+  deployment.
+
+* **Authentication request URL**: This is the address the tool sends its
+  response to the login initiation message, as specified in [Section
```

Comment:
The sentence is missing something. 

Is this correct?
... the address TO WHICH the tool sends its response ...

Reply:
You didn't react to this yet.